app/models/form/subsection.rb

```diff
@@ -62,4 +62,8 @@ def answered_questions(case_log)
   def unanswered_questions(case_log)
     applicable_questions(case_log) - answered_questions(case_log)
   end
+
+  def displayed_to_user?(case_log, question)
```

Comment:
Is this a little confusing? The question is still displayed to the user right? Just not the answer in the check answers page?

Reply:
So the 'what is the postcode?' question would not be displayed to the user if postcode known == "No". This would return false for the postcode, instead we would display 'postcode' question in the check answers page because `has_inferred_check_answers_value?` on line 55 would return true

Reply:
The only question `question.hidden_in_check_answers?` would currently return `True` for is the `Do you know the postcode?` question right? And that question is always displayed to the user?

Reply:
Oh sorry, I thought you were raising a different question.
Let's chat about it. I'm getting muddled in if statements